geoextent/lib/handleGeojson.py

```diff
@@ -0,0 +1,225 @@
+    returns the epsg code of the used coordinate reference system: type int, EPSG number of taken crs
+    ''' 
+    
+    def extractAfterKeyword(searchParam, gjsonContent):
+        ''' searches for the value fo the dict entry with keyword which is given as input \n
+        input "searchParam": type string, keyword for which is searched in the dict \n
+        input "gjsonContent": type dict, Content of geojson File
+        '''
+        if type(gjsonContent) == dict:
+            for keyContent, valueContent in gjsonContent.items():
+                if keyContent == searchParam:   
+                    extracted.append(valueContent)
+                if type(valueContent) == dict or type(valueContent) == list:
+                    extractAfterKeyword(searchParam, valueContent)
+        if type(gjsonContent) == list:
+            for element in gjsonContent:
+                extractAfterKeyword(searchParam, element)
+
+
+    try:
+        gjsonContent = pygeoj.load(filePath)
+        crsCode = gjsonContent.crs 
+        if not crsCode:
+            return hf.WGS84_EPSG_ID
+        else: 
+            for key, value in crsCode.items():
+                if key == "properties":
+                    try:
+                        if value["name"] == "urn:ogc:def:crs:OGC:2:84":
+                            return hf.WGS84_EPSG_ID
+                        elif value["name"]:
+                            splittedCrs = value["name"].split(":")
+                            for elem in splittedCrs:
+                                try:
+                                    if int(elem) is not None:
+                                        crsCode = int(elem)
+                                        return crsCode
+                                except:
+                                    pass
+                    except:
+                        pass
+                #formats like urn:ogc:def:crs:EPSG::25832
+                
+        return hf.WGS84_EPSG_ID
+    except:
+        gjsonContent = extractContentFromPath(filePath)
+    
+        #4326 is the standard epsg after http://wiki.geojson.org/GeoJSON_draft_version_6#Specification
+        crsCode = hf.WGS84_EPSG_ID
+        extracted = []
+        extractAfterKeyword("crs", gjsonContent)
+        if len(extracted) != 0:
+            if type(extracted[0]) == dict and "properties" in extracted[0] and "code" in extracted[0]["properties"]:
+                        crsCode = extracted[0]["properties"]["code"]
+        return crsCode
+
+
+
+def getTemporalExtent (filePath):
+    ''' extract time extent from json string \n
+    input "filePath": type string, file path to geojson File \n
+    returns the temporal extent of the file: type list, length = 2, both entries have the type dateTime, temporalExtent[0] <= temporalExtent[1]
+    '''
+    #type list, contains all dates
+    dateArray = []
+    
+    
+    def searchForTimeElements(gjsonContent):
```

Comment:
this function should be part of a `util.py` to be used by all `FormatHandler`s, see #28